.github/workflows/code_quality.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
-    uses: aws-deadline/.github/.github/workflows/reusable_python_build.yml@mainline
+    uses: AuxinJeron/.aws-deadline-github/.github/workflows/reusable_python_build.yml@ubuntu-pre-steps
```

Comment:
This will be updated while https://github.com/aws-deadline/.github/pull/24 has been approved